### PR TITLE
chore: switch openstack public endpoints to https

### PR DESCRIPTION
**Pull request by skrobul**

### CODE REVIEW

File: components/nova/aio-values.yaml [hunk 1]
```diff
@@ -18,6 +18,12 @@ endpoints:
       name: rabbitmq-server
     hosts:
       default: rabbitmq-nodes
+  compute:
+    port:
+      api:
```
Comment (cardoe) [inline components/nova/aio-values.yaml:23]:
```suggestion
      osapi:
```

I think nova uses that.

File: components/neutron/aio-values.yaml [hunk 2]
```diff
@@ -11,6 +11,13 @@ endpoints:
       name: rabbitmq-server
     hosts:
       default: rabbitmq-nodes
+  network:
+    port:
+      api:
```
Comment (cardoe) [inline components/neutron/aio-values.yaml:16]:
```suggestion
      server:
```

I think neutron uses this

**Review by cardoe — COMMENTED** [review]
I think you need these two.

File: components/neutron/aio-values.yaml (hunk 2 above)
Comment (skrobul) [reply components/neutron/aio-values.yaml:16]: hmm where did you see that?
[The openstack-helm values.yaml for neutron](https://opendev.org/openstack/openstack-helm/src/commit/8e5ad85fd09983e07ddc87de76e480b4ceca9120/neutron/values.yaml#L2535) seems to have it as 'api'

File: components/nova/aio-values.yaml (hunk 1 above)
Comment (skrobul) [reply components/nova/aio-values.yaml:23]: https://opendev.org/openstack/openstack-helm/src/commit/8e5ad85fd09983e07ddc87de76e480b4ceca9120/nova/values.yaml#L1880

**Review by cardoe — APPROVED** [review]

**Pull request merged**